### PR TITLE
[zh-cn]: Update translation of String.padEnd()

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#13975

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/string/padend/index.md [hunk 1]
```diff
@@ -5,35 +5,37 @@ slug: Web/JavaScript/Reference/Global_Objects/String/padEnd
...
+```js-nolint
 padEnd(targetLength)
 padEnd(targetLength, padString)
 ```
 
 ### 参数
 
 - `targetLength`
-  - : 当前字符串需要填充到的目标长度。如果这个数值小于当前字符串的长度，则返回当前字符串本身。
+  - : 当前 `str` 填充后的长度。如果该值小于或等于 `str.length`，则会直接返回当前字符串。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/string/padend/index.md:22]:
```suggestion
  - : 当前 `str` 填充后的长度。如果该值小于或等于 `str.length`，则会直接返回当前 `str`。
```

File: files/zh-cn/web/javascript/reference/global_objects/string/padend/index.md [hunk 2]
```diff
@@ -5,35 +5,37 @@ slug: Web/JavaScript/Reference/Global_Objects/String/padEnd
...
 - `padString` {{optional_inline}}
-  - : 填充字符串。如果字符串太长，使填充后的字符串长度超过了目标长度，则只保留最左侧的部分，其他部分会被截断。此参数的缺省值为 " "（U+0020）。
+  - : 用于填充当前 `str` 的字符串。如果 `padString` 太长，无法适应 `targetLength`，则会被截断：对于从左到右的语言，左侧的部分将会被保留；对于从右到左的语言，右侧的部分将会被保留。默认值为 “ ” (`U+0020`)。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/string/padend/index.md:24]:
```suggestion
  - : 用于填充当前 `str` 的字符串。如果 `padString` 太长，无法适应 `targetLength`，则会被截断：对于从左到右的语言，左侧的部分将会被保留；对于从右到左的语言，右侧的部分将会被保留。默认值为“ ” (`U+0020`)。
```

File: files/zh-cn/web/javascript/reference/global_objects/string/padend/index.md [hunk 3]
```diff
@@ -5,35 +5,37 @@ slug: Web/JavaScript/Reference/Global_Objects/String/padEnd
...
-  - : 当前字符串需要填充到的目标长度。如果这个数值小于当前字符串的长度，则返回当前字符串本身。
+  - : 当前 `str` 填充后的长度。如果该值小于或等于 `str.length`，则会直接返回当前 `str`。
 - `padString` {{optional_inline}}
-  - : 填充字符串。如果字符串太长，使填充后的字符串长度超过了目标长度，则只保留最左侧的部分，其他部分会被截断。此参数的缺省值为 " "（U+0020）。
+  - : 用于填充当前 `str` 的字符串。如果 `padString` 太长，无法适应 `targetLength`，则会被截断：对于从左到右的语言，左侧的部分将会被保留；对于从右到左的语言，右侧的部分将会被保留。默认值为“ ” (`U+0020`)。
 
 ### 返回值
 
-在原字符串末尾填充指定的填充字符串直到目标长度所形成的新字符串。
+在当前 `str` 末尾填充 `padString` 直到达到给定的 `targetLentgth` 所形成的 {{jsxref("String")}}。
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/global_objects/string/padend/index.md:28]:
```suggestion
在当前 `str` 末尾填充 `padString` 直到达到给定的 `targetLength` 所形成的 {{jsxref("String")}}。
```

**Review by jasonren0403 — CHANGES_REQUESTED** [review]
some suggestions

**Review by jasonren0403 — APPROVED** [review]
👍

**Pull request merged**